files/trapped-mix

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Just out of curiosity, why is this the only script that needs to be executed in sh instead of bash? Is it related to the `trap` behavior across command interpreters?

Reply:
Ah ok, good to know, thanks!